docs/user/about/index.rst

```diff
@@ -1,10 +1,24 @@
 About Read the Docs
 ===================
 
+
+.. toctree::
+   :caption: Additional commercial features
+   :hidden:
```

Comment:
How are users meant to find these pages, other than the sidebar? Are they linked anywhere else?

Reply:
A description list would be nice to introduce the content in this section!

Reply:
Caption in the TOC was also wrong...